### DISCUSSION TITLE
App published in the Microsoft Store for 5$

### DISCUSSION THREAD

**Discussion by YacineBoussoufa** [4gray/iptvnator#228] [General]
Hi, I'm here just to report that the app has been published by a certain "EliasFinnng" for 5$ on the Microsoft Store, I don't know if you're aware of this or it's a scam. Here's a photo and the [link](https://apps.microsoft.com/store/detail/iptvnator-smarters-player/9NJQZL486V43?hl=en-us&gl=us)

![image](https://user-images.githubusercontent.com/46266665/218875535-a805d9e3-dce1-4748-bba9-e6c4e157a990.png)

**Comment by 4gray** [comment]
Hey, thanks for reporting, it's scam.
IPTVnator has always been and remains a free open source application. 
Does anyone know what is the shortest way to report this issue in Microsoft Store? On a quick look at the page I couldn't find the link...

**Comment by YacineBoussoufa** [comment]
I found [this](https://www.microsoft.com/en-us/legal/intellectualproperty/infringement#:~:text=While%20using%20the%20appropriate%20online,to%20dmcaagnt%40microsoft.com.), the fastests way to report it, is to contact Microsoft via [dmcaagnt@microsoft.com](mailto:dmcaagnt@microsoft.com)
